hawc/apps/assessment/models.py

```diff
@@ -919,9 +919,30 @@ class Log(models.Model):
     class Meta:
         ordering = ("-created",)
 
+    def __str__(self) -> str:
+        if self.content_object is not None:
+            return str(self.content_object) + " Log"
+        if self.assessment is not None:
+            return str(self.assessment) + " Log"
+        return "Custom Log"
+
+    def get_absolute_url(self):
+        return reverse("assessment:log_detail", args=(self.id,))
+
     def get_api_url(self):
+        # TODO delete api stuff?
```

Comment:
@shapiromatron This method isn't called anywhere, so I can see it safely being deleted. Do you think we should go a step further and remove the log API altogether?

Reply:
Let's add a jira card to revisit in Q1 2022, sound ok?

Reply:
Done!